src/Api/API.php

```diff
@@ -2963,6 +2963,13 @@ protected function getNetworkPorts(
 
                 foreach ($netp_iterator as $data) {
                     if (isset($data['netport_id'])) {
+                        // append contact
+                        $npo = new NetworkPort();
+                        $oppositecontactID = $npo->getContact($data['netport_id']) ;
+                        if ($oppositecontactID) {
+                            $data['netport_oppositecontactID'] = $oppositecontactID ;
+                        }
```

Comment:
It is preferable to have similar keys in all JSON responses from a same endpoint.

```suggestion
                        } else {
                            $data['networkports_id_opposite'] = null;
                        }
```